### PR TITLE
Add fiber photometry

### PR DESCRIPTION
**Pull request by weiglszonja**
Add raw fiber photometry data from the binned .mat files.
The binned .mat files are a result of concatenation of PicoScope data after separating fluorescence due to 470 nm vs 405 nm illumination and re-binned to 100 Hz (see #5):

> Fluorescence collected during 405-nm or 470-nm illumination (20 time bins for each pulse of 405-nm or 470-nm excitation) was separated using the binary output from the waveform generator. For each transition period between illumination sources, five time bins were excluded to remove transition times. Traces were then re-binned to 100 Hz by averaging every 40 time bins for velocity and every 40 time bins for 405-nm and 470-nm fluores- cence traces (but including only 15 of 40 bins for each source: excluding 20 bins when the alternate source was on and five transition bins).

Using `ndx-photometry` (https://github.com/catalystneuro/ndx-photometry/pull/7)

The updated diagram:

azcorra2023_uml

### CODE REVIEW

File: src/dombeck_lab_to_nwb/azcorra2023/metadata/azcorra2023_fiber_photometry_metadata.yaml
Comment (weiglszonja) [inline src/dombeck_lab_to_nwb/azcorra2023/metadata/azcorra2023_fiber_photometry_metadata.yaml:1]: Just discussed today that we will provide metadata as list approach so I'll change the metadata for Fibers and other where I used the dict approach

**Comment by weiglszonja** [comment]
Stub is at [20200205-0001.nwb](https://eu-north-1.console.aws.amazon.com/s3/object/catalystneuro-processing?region=us-east-2&bucketType=general&prefix=dombeck-lab-to-nwb/20200205-0001.nwb)

File: src/dombeck_lab_to_nwb/azcorra2023/interfaces/azcorra2023_fiberphotometryinterface.py
Comment (weiglszonja) [inline src/dombeck_lab_to_nwb/azcorra2023/interfaces/azcorra2023_fiberphotometryinterface.py:24]: Note that if there are two fibers, the depth of the second fiber is not in the binned photometry file. I'm going to change this logic in https://github.com/catalystneuro/dombeck-lab-to-nwb/pull/10/commits/4a590011d12e7358b692dc879c349e2a159e0e90, to set it as a `conversion_option` instead init parameter.

**Review by alessandratrapani — COMMENTED** [review]
LGTM! I would wait after the meeting to approve because I have some questions about the representation of the wavelength.

**Comment by weiglszonja** [comment]
Yeah I think the metadata is probably going to change, looking at the notes for today.
I will have to ask these because I just couldn't find it in the [article](https://www.nature.com/articles/s41593-023-01401-9).

**Review by alessandratrapani — APPROVED** [review]

**Pull request merged**
